test/test_transforms_tensor.py

```diff
@@ -85,13 +84,12 @@ def _test_class_op(method, device, meth_kwargs=None, test_exact_match=True, **ma
     batch_tensors = _create_data_batch(height=23, width=34, channels=3, num_samples=4, device=device)
     _test_transform_vs_scripted_on_batch(f, scripted_fn, batch_tensors)
 
-    with get_tmp_dir() as tmp_dir:
-        scripted_fn.save(os.path.join(tmp_dir, f"t_{method.__name__}.pt"))
+    scripted_fn.save(os.path.join(tmp_dir, f"t_{method.__name__}.pt"))
```

Comment:
hm, the structure of the tests in this file is slightly awkward (for various historical reasons), and it makes it a bit cumbersome to pass the `tmpdir` fixture from the actual `test_...()` function all the way down to here.

So I would instead suggest to actually just rely on our custom `get_tmp_dir()` function for this and revert those bits. I'll try to find a cleaner solution in the future, potentially refactoring further those tests. This means that we should put back `get_tmp_dir()` in the `common_utils.py` file (sorry for the extra work!)

Reply:
Yes. That is also what I was wondering. I moved get_tmp_dir() back to common_utils.py but still replaced other get_tmp_dir() with tmpdir fixsure except the one in _test_class_op().